packages/wonder-blocks-labeled-field/src/components/__tests__/labeled-field.test.tsx

```diff
@@ -126,49 +145,6 @@ describe("LabeledField", () => {
         expect(error).toBeInTheDocument();
     });
 
-    it("LabeledField adds the correctly formatted id to label's htmlFor", () => {
-        // Arrange
-        const id = "exampleid";
-        const testId = "testid";
-
-        // Act
-        render(
-            <LabeledField
-                field={<TextField id="tf-1" value="" onChange={() => {}} />}
-                label="Label"
-                id={id}
-                testId={testId}
-            />,
-            defaultOptions,
-        );
-
-        // Assert
-        const label = screen.getByTestId(`${testId}-label`);
-        expect(label).toHaveAttribute("for", `${id}-field`);
-    });
-
-    it("LabeledField adds the correctly formatted id to error's id", () => {
```

Comment:
Removed since this is a duplicate for this test (the "error" case): https://github.com/Khan/wonder-blocks/blob/2b809b7d46afbf8bfab02559afb5778ad3723992/packages/wonder-blocks-labeled-field/src/components/__tests__/labeled-field.test.tsx#L260-L291